app/views/answers/_answer.html.erb

```diff
@@ -0,0 +1,80 @@
+<td id="vote-container-td">
+  <div id="vote-conatiner">
+      <div class="answer-vote-section">
+          <%if current_user && answer.already_voted?(current_user.id)%>
+
+            <%if answer.answer_up_or_down_vote(current_user.id) == "up"%>
```

Comment:
These could probably be a helper with an up/down parameter rather than repeating with a couple of chars difference.